consensus/moca/state.go

```diff
@@ -30,10 +30,13 @@ func (consensus *Consensus) startGettingNeighborConsensusState() {
 			if localConsensusHeight == 0 || localConsensusHeight+1 < majorityConsensusHeight {
 				if majorityConsensusHeight+1 > localLedgerHeight {
 					consensus.setNextConsensusHeight(majorityConsensusHeight + 1)
+					if consensus.localNode.GetSyncState() == pb.PersistFinished {
+						consensus.localNode.SetSyncState(pb.WaitForSyncing)
```

Comment:
Do we need set the lock before read the node state or set node state cause there is potential race case existed.

Reply:
We don't need to do it here, since the logic is sequential and it's the only place to change state back from persist finished